project/Dependencies.scala

```diff
@@ -34,7 +34,7 @@ object Dependencies {
   lazy val infoset = Seq(
     "org.jdom" % "jdom2" % "2.0.6",
     "com.fasterxml.woodstox" % "woodstox-core" % "5.1.0",
```

Comment:
Below is the only ACTUAL change in this PR.

Reply:
Anytime we update a dependency, we also need to update the LICENSE and NOTICE files (and also validate that the dependencies license didn't change). In this case I think just ``daffodil-cli/bin.NOTICE`` needs to be updated since we only distribute jackson code with the CLI, and it's Apache licensed.